text/0000-partial-turbofish.md

```diff
@@ -0,0 +1,314 @@
+- Feature Name: partial_turbofish
+- Start Date: 2017-10-16
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Assume a function (free, inherent, trait, ...)
+`fn turbo< $($tparam: ident),* )>(..) -> R` involving generic types
+`$($tparam: ident),*`. If while calling `turbo::< $($tconcrete: ty),* >(...)` a
+suffix of the applied types can be replaced with a list of `_`s of equal length,
+then the suffix may be omitted entirely. A shorter suffix may be chosen at will.
+This also applies to *turbofish*ing types (structs, enums, ..), i.e:
```

Comment:
This `*` in the middle of the word seems to be messing up GH's markdown -- at least in the preview. Can you remove it?

Reply:
Removed =)